### PR TITLE
Feature/add chainlink

### PR DESCRIPTION
**Pull request by melchermaxwell**
## Description

Add Chainlink to the Scroll Developer tools

## Changes

-added chainlink.mdx with relevant info

### CODE REVIEW

**Comment by dghelm** [comment]
Glad to see Chainlink added here! Wondering if we could provide useful information for Scroll devs, since only a subset of Chainlinks docs are relevant to developers.

Could we provide LINK token address, Sequencer Uptime Feed address (or direct link) along with a pricefeed link for Scroll?

File: src/content/tools/en/chainlink.mdx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: "Chainlink"
+category: ["Oracle", "Bridge"]
+excerpt: ""
+logo: { src: "https://pbs.twimg.com/profile_images/1800426318099595264/N7yf_kOD_400x400.jpg", alt: "Chainlink Logo" }
+website: "https://chain.link/"
+network: ["Mainnet", "Testnet"]
+---
+
+You can find contract addresses for Chainlink on [Scroll Mainnet](https://docs.chain.link/).
```
Comment (thedriftofwords) [inline src/content/tools/en/chainlink.mdx]:
```suggestion
You can find contract addresses for Chainlink for Scroll Mainnet and Scroll Sepolia testnet:

- [LINK Token Contracts for Scroll](https://docs.chain.link/resources/link-token-contracts#scroll)
- [L2 Sequencer Data Feeds](https://docs.chain.link/data-feeds/l2-sequencer-feeds#available-networks)
- [Price Feed Addresses for Scroll](https://docs.chain.link/data-feeds/price-feeds/addresses?network=scroll&page=1)


```

**Review by thedriftofwords — CHANGES_REQUESTED** [review]

**Comment by thedriftofwords** [comment]
@dghelm thanks for the suggestions. I've added links to the sections of the Chainlink docs that contain the info you asked for. We prefer to use docs links instead of including the addresses directly, so that you have the link to the latest info in case anything changes in the future.

File: src/content/tools/en/chainlink.mdx (hunk 1 above)
Comment (emilyJLin95) [reply src/content/tools/en/chainlink.mdx]: Can you merge in this change? And hten I can approve

**Review by emilyJLin95 — APPROVED** [review]
LGTM

**Review by thedriftofwords — APPROVED** [review]

**Review by ChuhanJin — APPROVED** [review]

**Pull request merged**